### PR TITLE
Add docs for integration with ZEPL login and 0.7.x

### PR DESCRIPTION
**Pull request by khalidhuseynov**

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
Looks great!
To let users know there is version-by-version info, it would be better to generate subsection to left navbar. 

restructure

Let me push a commit for this in here if it looks okay.

**Comment by khalidhuseynov** [comment]
@AhyoungRyu yeah sure, feel free to push :)

**Comment by khalidhuseynov** [comment]
@AhyoungRyu also could you try to follow steps and login? I just tried with Zeppelin binaries for versions `0.7.0` and `0.7.1`, could login on `0.7.0` but was having error in `0.7.1`

**Comment by AhyoungRyu** [comment]
@khalidhuseynov okay 👌🏼

**Comment by khalidhuseynov** [comment]
@AhyoungRyu I actually managed with `0.7.1` as well, the reason was that didn't change some config in `zeppelin-env.sh`. but still would be nice if you try too

**Comment by AhyoungRyu** [comment]
@khalidhuseynov  I was about to let you know I could successfully connected Zepl with 0.7.1 actually kk

File: docs/zeppelin_integration.md [hunk 1]
```diff
@@ -68,21 +68,61 @@ export ZEPPELINHUB_USER_KEY="YOUR_USER_KEY"
...
+```
+<br/>
+<span class="version-font"> Zeppelin-0.7.x multiple users on Zeppelin server</span>
+
+```sh
+export ZEPPELIN_NOTEBOOK_STORAGE="org.apache.zeppelin.notebook.repo.VFSNotebookRepo, org.apache.zeppelin.notebook.repo.zeppelinhub.ZeppelinHubRepo"
+export ZEPPELINHUB_API_ADDRESS="ADDRESS_OF_ZEPPELINHUB_SERVICE" (e.g. https://www.zepl.com)
 ```
-As you can see, only `ZEPPELIN_NOTEBOOK_STORAGE` was updated on **Zeppelin-0.6.X**.
+As you can see, you don't need `ZEPPELINHUB_API_TOKEN` in this case, and you can utilize login with your ZEPL account, as described in the next section. In this case your token will be automatically recognized from your `instance` information and default one will be loaded. Note that if you're following this scenario, you'll need to complete the [next section](link to ## Login to Apache Zeppelin 0.7.x with ZEPL credentials) as well in order to login with your ZEPL credentials.
```
Comment (khalidhuseynov) [inline docs/zeppelin_integration.md]: @AhyoungRyu could you replace here link to real path for the next section in `[next section](link)`

**Comment by khalidhuseynov** [comment]
added changes including recommendation

File: docs/zeppelin_integration.md (hunk 1 above)
Comment (AhyoungRyu) [reply docs/zeppelin_integration.md]: Addressed in dac61939622cec849d8098ebecce534a5633290f :)

**Comment by AhyoungRyu** [comment]
@khalidhuseynov Looks good!
Regarding the left navbar substructure, seems Mkdocs doesn't support auto toc generator from level 3 header. Let me find another workaround for this.

**Review by AhyoungRyu — APPROVED** [review]

**Comment by khalidhuseynov** [comment]
then i'll merge it and let's handle it separately. one way would be to separate it completely for 0.5.x, 0.6.x and 0.7.x, or maybe there's better way

**Pull request merged**